README.md

```diff
@@ -30,62 +30,57 @@ $ git clone git@github.com:phoenix-dataplane/phoenix.git --recursive
+`cmake` available on your system.
+Additionally, you need to have `rustup` and `cargo-make` installed.
+For Ubuntu 22.04, you can use the following commands:
 ```
-# apt install libclang-dev libnuma-dev librdmacm-dev libibverbs-dev protobuf-compiler cmake
+$ sudo apt-get update
+$ sudo apt install libclang-dev libnuma-dev librdmacm-dev libibverbs-dev protobuf-compiler cmake
 $ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 $ cargo install cargo-make
 ```
 
-Alternatively, if you already have VS Code and Docker installed, you can click the badge above or [here](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/phoenix-dataplane/phoenix) to get started.
+Alternatively, if you already have VS Code and Docker installed, click the badge above or [here](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/phoenix-dataplane/phoenix) to start.
 
-3. Build and run phoenixos service.
+3. Build and run PhoenixOS service.
 ```bash
 $ cargo make
 ```
-`cargo make` without specifying a build target will build the default
-target, which is "dev-test-flow" by default. You can inspect and
-customize the stages of this default workflow in `Makefile.toml`.
+By default, cargo make will build the `dev-test-flow` target. You can inspect and
+customize the stages of this workflow in `Makefile.toml`. Use `cargo make run-phoenixos` to start the service after building.
 
-Optionally, you can manually execute each step in the dev-test-flow.
+You can also manually execute each step in the dev-test-flow.
 
-Note, once you have finished building, you can use
-`cargo make run-phoenixos` to start the service.
-
-Without any plugin, phoenixos itself is just an empty control plane. Next,
+PhoenixOS without any plugins is just an empty control plane. Next,
 you can build and load some useful plugins and run a few user applications.
 
-### Building mRPC
+### Building and Running mRPC
 mRPC is the first experimental feature on Phoenix.
-You can simply change directory to `experimental/mrpc` to build, 
-deploy mRPC plugins, and run phoenixos in one command.
+To build and deploy mRPC plugins, and run PhoenixOS, follow these steps:
 
 ```bash
 $ cd experimental/mrpc
 $ cat load-mrpc-plugins.toml >> ../../phoenix.toml
 $ cargo make
 ```
 
-For inter-host RPC in the following `rpc_hello` example, you may need at
+<!-- For inter-host RPC in the following `rpc_hello` example, you may need at
 least two machines. However, you can still run the client and server on the same
 machine, communicating through the same instance of phoenixos.
-Choose whichever test scenario that is most suitable for you.
+Choose whichever test scenario that is most suitable for you. -->
 
-To begin with, make sure exactly one instance of phoenixos is running on all servers,
+Ensure that exactly one instance of PhoenixOS is running on all servers.
```

Comment:
I just realized that this sentence could be interpretted differently. Maybe change it to:
Ensure that exactly one instance of PhoenixOS is running on each server.